R/ecr_metaepoch.R

```diff
@@ -0,0 +1,43 @@
+#' Function that runs one ecr metaepoch.
+#'
+#' @param config_ecr - list of ecr::ecr params
+#'
+#' @return list with named fields: solution, population, value
+#' @export
+#'
+#' @examples
+ecr_metaepoch <- function(config_ecr) {
+  function(fitness, suggestions, lower, upper, tree_level) {
+    config <- config_ecr[[tree_level]]
+    legal_passed_param_names <- Filter(function(name) {
+      name %in% methods::formalArgs(ecr::ecr)
+    }, names(config))
+    params <- list(
+      # TODO/Review: Lambda?
+      "mu" = nrow(suggestions),
+      "mutator" = ecr::setup(ecr::mutGauss, lower = lower, upper = upper),
+      "lambda" = 1L)
```

Comment:
Też bym to chyba tak ustawił. Nie do końca rozumiem jak działa ta lambda, no ale z opisu wynikałoby jakby miała odpowiadać za jakieś zwiększenie populacji, czego byśmy raczej nie chcieli.

![image](https://user-images.githubusercontent.com/24687031/145718420-4270abdc-061c-45ae-8002-a97018228bc1.png)